matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/model/RoomSummary.kt

```diff
@@ -60,7 +61,8 @@ data class RoomSummary(
         val roomType: String? = null,
         val spaceParents: List<SpaceParentInfo>? = null,
         val spaceChildren: List<SpaceChildInfo>? = null,
-        val flattenParentIds: List<String> = emptyList()
+        val flattenParentIds: List<String> = emptyList(),
+        val userPresence: UserPresence? = null
```

Comment:
I guess this is only for DM rooms? Maybe the name should reflect

Reply:
Also please move it just after `directUserId`